setup.py

```diff
@@ -138,7 +150,7 @@ def getpyexts():
     if IS_WIN:
         libraries_plat = ['daal_core_dll']
     else:
-        if dpcpp:
+        if dal_build_version == (2021,106):
```

Comment:
I suggest to move this behavior to another place like it is done here (branch, is not merged yet):
https://github.com/IntelPython/daal4py/blob/feature/sycl-sklearn/setup.py#L95
these changes are in this commit
https://github.com/IntelPython/daal4py/commit/8e6e84270ce10a2449b42595f5706816759b3d80

Reply:
done